extensions/ql-vscode/src/helpers.ts

```diff
@@ -43,6 +43,12 @@ export const tmpDirDisposal = {
   },
 };
 
+interface ShowAndLogOptions {
+  outputLogger?: OutputChannelLogger;
+  items?: string[];
+  fullMessage?: string;
```

Comment:
Can we also add documentation for the properties here? We could even do this instead of in the method documentation, that avoids having to repeat the documentation in three different methods.

Reply:
Good idea. I'll move the documentation to the fields here instead of on the methods. I can still include a link to the type from the method documentation.